src/App.js

```diff
@@ -3,28 +3,40 @@ import { useEffect, useState } from 'react';
 import axios from 'axios';
 import { nanoid } from 'nanoid';
 import styled from 'styled-components';
+import Header from './components/Header.js';
 import Navigation from './components/Navigation.js';
 import AllPhrases from './components/pages/AllPhrases.js';
 import AddPhrases from './components/pages/AddPhrases.js';
+import CreateKidsPage from './components/pages/CreateKidsPage.js';
 import FavoritePhrases from './components/pages/FavoritePhrases.js';
 import breakpoint from './components/commons/breakpoints.js';
 
 const PRESET = process.env.REACT_APP_CLOUDINARY_PRESET;
 const CLOUDNAME = process.env.REACT_APP_CLOUDINARY_CLOUDNAME;
 
+const initialPhrase = {
+  date: '25. März 2022',
+  id: 'PYaHT9ymtyVHW2tCollee',
+  isBookmarked: false,
+  photo: 'sybgbgokbdhkutb2xbx5',
+  text: 'ahh kackscheisse!',
+};
```

Comment:
I would probably save this as an array here already:
```suggestion
const initialPhrases = [
 {
    date: '25. März 2022',
    id: 'PYaHT9ymtyVHW2tCollee',
    isBookmarked: false,
    photo: 'sybgbgokbdhkutb2xbx5',
    text: 'ahh kackscheisse!',
  }
];
```

Reply:
Hey Lene! thx a lot for reviewing ;)